docs/animations/AnimationSet.md

```diff
@@ -35,6 +38,12 @@ anim.SetDurationForAll(2500);
 anim.SetDelay(250);
 anim.Start();
 ```
+```vb
```

Comment:
The XAML and C# sections should be removed since each code block already specifies the language. 

![image](https://user-images.githubusercontent.com/711864/38783711-bbb82734-40ba-11e8-9ec0-3a9f27b690a4.png)


@Vijay-Nirmal, should this be done in your PR?

Reply:
On it

Reply:
Done